### ISSUE TITLE
support `make dist` and `make distcheck` - make release tarballs

### ISSUE THREAD

**Issue by sofar** [01org/isa-l#4]
Please consider making "release tarballs" and publishing them with every release.

The canonical way of making these is using ```make dist```. Once that creates a packaged ```$project-$version.tar.$ext``` tarball, you can upload these to github by going to ```https://github.com/01org/isa-l/releases```, clicking on the tag and/or ```edit release``` and then drag-and-drop your tarball release file.

Currently, ```make dist``` and ```make distcheck``` fail, so it's not possible to create autotools release files. The error is:

```
$ make distcheck
make --no-print-directory dist-xz am__post_remove_distdir='@:'
Building isa-l.h
make[1]: *** No rule to make target 'About_bsd.txt', needed by 'distdir'.  Stop.
Makefile:2024: recipe for target 'dist' failed
make: *** [dist] Error 2
```

I assume it's because the file mentioned was not added to git, and thus this fails.

**Comment by sofar** [comment]
FYI, here's how we're doing this for ClearLinux projects:

https://github.com/clearlinux/clr-cloud-init/releases

**Comment by gbtucker** [comment]
Fixed `make dist` in fce681a

**Comment by sofar** [comment]
Thanks!
